### PR TITLE
feat(EdrManagementApi): open transfer refactor

### LINKED ISSUE #346: Refactor `TransferOpenRequest` to `NegotiateEdrRequest`
## Describe the solution you'd like

Refactor the old temporary name of "open transfer" to EDR management.

TransferOpenRequest sould be renamed into `NegotiateEdrRequest` into the path `/adapter/edrs`
Move also the API to a JSON-LD based format, following the new EDC migration to JSON-LD

### PR DESCRIPTION
**Pull request by wolf4ood**
## WHAT

`TransferOpenRequest` has been renamed into `NegotiateEdrRequest` 
with the API path for starting a new EDR negotiation as  `/adapter/edrs`

The API as also been  moved to JSON-LD based format.

## WHY

consistency and compliance with the latest EDC version

## FURTHER NOTES

In this PR also  there is a migration to the latest version of EDC `0.0.1-milestone-9` and the new `dsp` protocol.
Subsequent PRs  should be made in order to clean the not working code due the migration to `dsp` protocol

Closes #346

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 format.version = "1.1"
 
 [versions]
-edc = "0.0.1-20230509-preview-SNAPSHOT"
+edc = "0.0.1-20230516-SNAPSHOT"
```
Comment (ndr-brt) [inline gradle/libs.versions.toml]: `0.0.1-milestone-9` could be used

**Review by ndr-brt — APPROVED** [review]
some nits, but LGTM overall :rocket:

File: gradle/libs.versions.toml (hunk 1 above)
Comment (wolf4ood) [reply gradle/libs.versions.toml]: Just did

File: edc-tests/e2e-tests/src/test/java/org/eclipse/tractusx/edc/helpers/CatalogHelperFunctions.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import static org.eclipse.edc.jsonld.spi.PropertyAndTypeNames.ODRL_POLICY_ATTRIBUTE;
+import static org.eclipse.edc.spi.CoreConstants.EDC_NAMESPACE;
+
+public class CatalogHelperFunctions {
+
+
+    public static JsonObject createCatalogRequest(JsonObject query, String dspEndpoint) {
+        var jsonBuilder = Json.createObjectBuilder();
+        jsonBuilder.add("@type", "CatalogRequest");
+        jsonBuilder.add(EDC_NAMESPACE + "providerUrl", dspEndpoint);
```
[the remark this answers is not in the document]
Comment (wolf4ood) [reply edc-tests/e2e-tests/src/test/java/org/eclipse/tractusx/edc/helpers/CatalogHelperFunctions.java:34]: @ndr-brt  I've raise an issue there #373 for this

**Pull request merged**